### PR TITLE
Update to allow video quality grouping for browser fallback media type.

### PR DESCRIPTION
**Pull request by Gungrave223**
Note... I'm not a js guy...so this is the best I can do with my current knowledge...

~~I don't know what "t.mediaFiles" is...I'm guessing it as something to do with the main mejs file...so to avoid breaking anything I just created a separate branch in the code...~~

If there is something that I can improve on let me and I'll update.

### CODE REVIEW

**Comment by rafa8626** [comment]
Thanks for this PR. I'll review once I have some free time

**Comment by Gungrave223** [comment]
Ok...let me know what issue(s) you find and I'll fix asap

**Comment by rafa8626** [comment]
@Gungrave223 Overall, great job! I left some comments there. And I'm about to testing the plugin now. So I'll let you know if I find anything functional that might not be working

**Comment by rafa8626** [comment]
Also make sure you resolve the conflicts with your PR

**Comment by rafa8626** [comment]
@Gungrave223 did you ever check if this will work with IE 11?

**Comment by Gungrave223** [comment]
No I only test in edge....Will test in IE 11 when I get home later though for any possible issues..

File: src/quality/quality.js [hunk 1]
```diff
@@ -153,7 +153,7 @@ Object.assign(MediaElementPlayer.prototype, {
 		for (let i = 0, total = radios.length; i < total; i++) {
 			const radio = radios[i];
 			radio.disabled = false;
-			radio.addEventListener('click', function() {
+			radio.addEventListener('change', function() {
```
[the remark this answers is not in the document]
Comment (Gungrave223) [reply src/quality/quality.js]: Yes...I could be wrong but internally I believe radio button uses click event to update the state of each button...so when you click to change the quality, what I notice was the event being called on each button in the group which was not the indented behavior.

File: src/quality/quality.js [hunk 2]
```diff
@@ -36,9 +36,11 @@ Object.assign(MediaElementPlayer.prototype, {
 	buildquality (player, controls, layers, media) {
 
 		const
-			t = this,
-			qualities = [],
-			children = t.mediaFiles ? t.mediaFiles : t.node.children
+				t = this,
```
[the remark this answers is not in the document]
Comment (Gungrave223) [reply src/quality/quality.js]: Hey not sure what you use as your editor but I use intellij and the indentation in that is 4 not 8, in order to maintain your current file format.

File: src/quality/quality.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (rafa8626) [reply src/quality/quality.js]: I see. As long as it works cross-browser I’m ok with this

**Comment by rafa8626** [comment]
I think it is solved in the formatting side; the only question is about support for IE11, and that includes Win7 platforms since they are still popular

**Comment by rafa8626** [comment]
Any updates on this?

**Comment by Gungrave223** [comment]
Sorry got super busy...

I was able to test and confirm that it works on Win 10 & 7 IE 11....I had to use browserstack to test Win 7 IE 11.

**Comment by rafa8626** [comment]
Perfect just keep me posted so I can test once more and merge it

**Comment by rafa8626** [comment]
PR merged. Thanks for your work on this

**Pull request merged**